contracts/ParsiqPancakeConverter.sol

```diff
@@ -0,0 +1,97 @@
+// SPDX-License-Identifier: MIT
+
+// IQ Protocol. Risk-free collateral-less utility renting
+// https://iq.space/docs/iq-yellow-paper.pdf
+// (C) Blockvis & PARSIQ
+// 🖖 Stake strong!
+
+pragma solidity 0.8.4;
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "./interfaces/IConverter.sol";
+import "./libs/Errors.sol";
+import "./libs/uniswap-v2/IUniswapV2Router02.sol";
+import "./libs/uniswap-v2/IUniswapV2Pair.sol";
+import "./libs/uniswap-v2/IUniswapV2Factory.sol";
+
+/**
+ * Pancakeswap converter for estimating token prices.
+ */
+contract ParsiqPancakeConverter is IConverter {
+    IUniswapV2Pair public swapPair;
+    IUniswapV2Router02 private _uniswapRouter;
```

Comment:
`immutable`

Reply:
This could not be made as `immutable` because methods get called on the `_uniswapRouter` variable inside the constructor.

Reply:
You can call on `uniswapRouter` parameter

Reply:
Thanks for the tip, updated the code :+1: